sgx/src/attestation_types/ti.rs

```diff
@@ -21,6 +22,31 @@ pub struct TargetInfo {
     reserved2: [u64; 25],
 }
 
+#[repr(C, align(128))]
+/// Pass information from the source enclave to the target enclave
+pub struct ReportData(pub [u8; 64]);
+
+#[cfg(feature = "asm")]
+impl TargetInfo {
+    /// Generate a report to the specified target with the included data.
+    pub fn get_report(&self, data: &ReportData) -> report::Report {
+        let mut report = core::mem::MaybeUninit::<report::Report>::uninit();
+        const EREPORT: usize = 0;
+
+        unsafe {
+            asm!(
+            "enclu",
+            in("rax") EREPORT,
+            in("rbx") self,
+            in("rcx") data.0.as_ptr(),
+            in("rdx") report.as_mut_ptr(),
+            );
```

Comment:
```suggestion
            asm!(
                "enclu",
                in("rax") EREPORT,
                in("rbx") self,
                in("rcx") data.0.as_ptr(),
                in("rdx") report.as_mut_ptr(),
            );
```

Reply:
@npmccallum  
cargo fmt is formatting it that way (without tabs)
Should I try committing without doing cargo fmt in sgx crate?

Reply:
No. We'll accept whatever cargo fmt does.